build.gradle

```diff
@@ -23,12 +23,14 @@ apply plugin: "com.gradle.plugin-publish"
 
 repositories {
     jcenter()
+    maven { url 'https://jitpack.io' }
 }
 
 dependencies {
     compile gradleApi()
     compile 'org.codehaus.groovy:groovy-all:2.4.10'
-    compile 'com.github.groovy-wslite:groovy-wslite:1.1.3'
+    // compile 'com.github.groovy-wslite:groovy-wslite:1.1.3'
```

Comment:
Not sure why we want to use the latest wslite here. The 1.1.x releases are stable, whereas I had instability issues with 2.x. This is why I've chosen to stick to the old stable version. I also don't like depending on snapshots as this causes instability in builds.